crates/aptos-rosetta/src/client.rs

```diff
@@ -384,33 +385,31 @@ impl RosettaClient {
         let mut signatures = Vec::new();
         let mut signers: Vec<AccountIdentifier> = Vec::new();
 
-        // Go through all payloads, and sign them accordingly
-        for payload in unsigned_response.payloads {
-            // Payloads must have a signer and an associated key
-            if let Some(ref account) = payload.account_identifier {
-                let address = account.account_address()?;
-
-                if let Some(private_key) = keys.get(&address) {
-                    // Sign the message
-                    let signing_bytes = hex::decode(&payload.hex_bytes)?;
-                    let txn_signature = private_key.sign_arbitrary_message(&signing_bytes);
-
-                    signers.push(address.into());
-                    signatures.push(Signature {
-                        signing_payload: payload,
-                        public_key: private_key.public_key().try_into()?,
-                        signature_type: SignatureType::Ed25519,
-                        hex_bytes: txn_signature.to_encoded_string()?,
-                    })
-                } else {
-                    return Err(anyhow!(
-                        "Address in payload doesn't have an associated key {}",
-                        address
-                    ));
-                }
-            } else {
-                return Err(anyhow!("No account in payload to sign!"));
-            }
+        // Sign the unsigned transaction
```

Comment:
Do we have a test for all of this stuff? To verify that the signatures are legit at the end of all this.

Reply:
This client is only used in a test :)

It's just a shared client for me to use for a testing CLI as well as the tests.

The `test_block` rosetta test, tests all of this (and was failing when I did something wrong)